azurerm/data_source_monitor_action_group_test.go

```diff
@@ -170,7 +220,41 @@ resource "azurerm_monitor_action_group" "test" {
   webhook_receiver {
     name        = "callmybackupapi"
     service_uri = "https://backup.example.com/warning"
+	}
+
+	automation_runbook_receiver {
+		name = "action_name_1"
+		automation_account_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001"
+		runbook_name = "my runbook"
+		webhook_resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001/webhooks/webhook_alert"
+		is_global_runbook = true
+		service_uri = "https://s13events.azure-automation.net/webhooks?token=randomtoken"
+	}
+
+	voice_receiver {
+    name         = "oncallmsg"
+    country_code = "1"
+    phone_number = "1231231234"
   }
+
+  voice_receiver {
+    name         = "remotesupport"
+    country_code = "86"
+    phone_number = "13888888888"
+	}
+
+	logic_app_receiver {
+		name = "logicappaction"
+		resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg-logicapp/providers/Microsoft.Logic/workflows/logicapp"
+		callback_url = "http://test-host:100/workflows/fb9c8d79b15f41ce9b12861862f43546/versions/08587100027316071865/triggers/manualTrigger/paths/invoke?api-version=2015-08-01-preview&sp=%%2Fversions%%2F08587100027316071865%%2Ftriggers%%2FmanualTrigger%%2Frun&sv=1.0&sig=IxEQ_ygZf6WNEQCbjV0Vs6p6Y4DyNEJVAa86U5B4xhk"
+	}
+
+	azure_function_receiver {
+		name = "funcaction"
+		function_app_resource_id = ""
```

Comment:
could we provision [a function app](https://www.terraform.io/docs/providers/azurerm/r/function_app.html) dynamically too and reference it's ID here?